tests/functional/test_inference_per_framework.py

```diff
@@ -399,6 +406,7 @@ def test_custom_models_with_drum_prediction_server(
         ],
     )
     @pytest.mark.parametrize("pass_args_as_env_vars", [True])
+    @pytest.mark.parametrize("max_workers", [1])
```

Comment:
Why parametrize when there is only one param?

Reply:
To keep it consistent with @pytest.mark.parametrize("pass_args_as_env_vars", [True]). As we reuse same function in different tests. I think the test is too much parameterise but it works